### ISSUE TITLE
[DetailsList] Announcing wrong number of rows for the DetailsList grid

### ISSUE THREAD

**Issue by harshil93** [OfficeDev/office-ui-fabric-react#3067]
Are you willing to submit a PR to fix? Yes

Requested priority: High

Products/sites affected: VSTS

#### Describe the issue:

Wrong number of rows are announced for details list because aria-rowcount is set to number of rows + 1 ( probably it is including header also) 

#### Actual behavior:

Announcing 501 rows

#### Expected behavior:

Should announce 500 rows

### If applicable, please provide a codepen repro:

Can be repored on official fabric react website

![image](https://user-images.githubusercontent.com/1177554/31326506-95bf409e-ace5-11e7-8cc7-2a6a681fb717.png)

**Comment by erichdev** [comment]
`aria-rowcount={ (isHeaderVisible ? 1 : 0) + (items ? items.length : 0) }`
The header is being counted only when it's visible. Should the header never be counted? Seems intentional in the code, but not sure what the standard is

**Comment by RegexEmpire** [comment]
We ran into this as well. The keyboard behavior does not allow you to navigate to the header, so this seems wrong to us.

**Comment by dwilsonmsft** [comment]
I don't think this is a bug. Although the Fabric doesn't make the header keyboard navigable, screenreader users can use the screenreader to navigate to the header and read its contents. Check [the examples from ARIA here](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/grid/dataGrids.html), where Windows Narrator reads each grid as having n+1 rows, and example 3 explicitly sets aria-rowindex on the header to 1 and to 2 on the first data row. Thus aria-rowcount should be n+1 whenever a header is displayed.

**Comment by michaelangotti** [comment]
This issue sounds like it is resolved. Closing.
